docs/tracing.md

```diff
@@ -33,16 +33,35 @@ outputting a `Trace` level log when entering and exiting every function.
 It is disabled by default at compile-time. Tracing functionality has no
 impact on the release binary.
 
+You can use `cargo run --bin clippy-tracing --` to build and run the
+latest version in the repo or you can run
+`cargo install --path src/clippy-tracing` to install the binary then use
+`clippy-tracing` to run this binary.
+
+You can run `clippy-tracing --help` for help.
+
 To enable tracing in Firecracker, add instrumentation with:
 
 ```
 clippy-tracing \
 --action fix \
 --path ./src \
---exclude vmm_config/logger.rs,virtio/gen,bindings.rs,net/gen,benches,logger/,signal_handler.rs,time.rs
+--exclude \
```

Comment:
could add indentation for better readability like
```bash
clippy-tracing \
  --action fix \
  --path ./src \
  --exclude \
```

Reply:
I personally prefer it without the indents since the additional spaces are not ignored. With 1 space if it's copied and placed into 1 line it looks normal, with 4 space it can look unusual.

Reply:
> additional spaces are not ignored

not sure I understand that. Do you mean if we add spaces they are not always rendered?

> with 4 space it can look unusual.

I think 2 spaces would serve both purposes?

Reply:
> This in my experience is not always consistent using the terminal, some terminals will format it as:

It doesn't look like a big concern to me as long as the command is still syntactically valid.